lib/src/kuzzle/request.dart

```diff
@@ -200,7 +203,9 @@ class KuzzleRequest {
     if (source != null) {
       map['source'] = source;
     }
-
+    if (defaults != null) {
+      map['defaults'] = defaults;
```

Comment:
I'm not sure to understand what this is suppose to do but `defaults` is part of the body and it looks like you put it in the arguments